extensions/interactions/InteractiveMap/directives/oppia-interactive-interactive-map.component.ts

```diff
@@ -0,0 +1,195 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
2021*

Reply:
Technically it is just a git rename where git didn't realize the rename due to the sheer number of changes.